generators/server/templates/src/main/java/package/config/_JHipsterProperties.java

```diff
@@ -228,22 +228,36 @@ public void setFrom(String from) {
 
         private final RememberMe rememberMe = new RememberMe();
         <%_ } _%>
-        <%_ if (authenticationType == 'oauth2' || authenticationType == 'jwt' || authenticationType == 'uaa') { _%>
+
+        <%_ if((applicationType === 'microservice' || applicationType === 'uaa') && authenticationType === 'uaa') { _%>
+
+        private final ClientAuthorization clientAuthorization = new ClientAuthorization();
+
```

Comment:
using ClientCredentialsResourceDetails (which is parent of LoadBalancedResourceDetails...)